appyters/RNA_Level_Functional_Predictions/appyter.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "https://raw.githubusercontent.com/MaayanLab/appyter-catalog/master/schema/appyter-validator.json",
+  "name": "RNA_Level_Functional_Predictions",
+  "title": "RNA level functional predictions",
+  "version": "0.0.3",
+  "description": "",
```

Comment:
Please add a short description about your appyter here to appear on the card for your appyter.